reinforcement_learning/bindings/cs/rl.net.native/rl.net.global.cc

```diff
@@ -0,0 +1,3 @@
+#include "rl.net.global.h"
+
+#include <iostream>
```

Comment:
Why is iostream added here?

Reply:
This used to have SayHello as a first-pass "binding work" test. Can probably be removed outright at this point.